RenderSupport.ls

```diff
@@ -0,0 +1,676 @@
+// =================================================================================================
+//
+//	Starling Framework
+//	Copyright 2011-2014 Gamua. All Rights Reserved.
+//
+//	This program is free software. You can redistribute and/or modify it
+//	in accordance with the terms of the accompanying license agreement.
+//
+// =================================================================================================
+
+package loom2d.core
+{
+    import loom2d.display.BlendMode;
+    import loom2d.display.DisplayObject;
+    import loom2d.display.Quad;
+    import loom2d.display.QuadBatch;
+    import loom2d.display.Stage;
+    import loom2d.math.Matrix;
+    import loom2d.math.Point;
+    import loom2d.math.Rectangle;
+    import loom2d.textures.Texture;
+    import loom2d.textures.TextureSmoothing;
+
+    /** A class that contains helper methods simplifying Stage3D rendering.
+     *
+     *  A RenderSupport instance is passed to any "render" method of display objects. 
+     *  It allows manipulation of the current transformation matrix (similar to the matrix 
+     *  manipulation methods of OpenGL 1.x) and other helper methods.
+     */
+    public class RenderSupport
```

Comment:
Lol, accidentally added it in along with others, good catch, thanks! If we implemented it, it might help with easier porting or it might open an abstraction way to hell. I decided against using it as we were doing fine without it anyway.